app/src/main/java/com/amaze/filemanager/crashreport/ErrorActivity.java

```diff
@@ -147,7 +147,13 @@ public static void reportError(
       final Context context, final CrashReportData report, final ErrorInfo errorInfo) {
     System.out.println("ErrorActivity reportError");
     final String[] el = new String[] {report.getString(ReportField.STACK_TRACE)};
-
+    // Add this to try figure out what happened when stacktrace is sent to acra.
+    // Hope this will be useful for build failures...
+    if (BuildConfig.DEBUG) {
+      for(String line: el) {
+        System.out.println(line);
```

Comment:
`System.out.print` should not be used in Android. Please use standard log, or add the info directly into the acra report.